project-koku/koku
# Remove purge_line_item_only for azure.

During the QA phase of: https://github.com/project-koku/koku/issues/1616

@lcouzens discovered there was a slight regression regarding ocp on azure. After debugging with @adberglund we discovered that azure doesn't actually have a `line_item` table all the data for azure comes in with a daily summary and removing the raw line item daily summary was causing some weird cascading delete issue. Therefore, we decided to remove the purge line item only feature for any azure provider. 

### how to test
```
make docker-reinitdb-with-providers; sleep 2; make load-test-customer-data start=2020-01-01
```
Wait for celery to finish processing and you should now see:
```
koku_worker       | [2020-02-27 16:44:40,845] INFO d0555734-f8ed-46a0-95ac-aa8549328dfc Azure-local has no line item data to be be removed.
```
in the logs
